clients/js/src/index.ts

```diff
@@ -385,14 +558,18 @@ export class Collection {
       }
     }
 
+    // backend expects None if metadatas is undefined
+    if (metadatas !== undefined) metadatas = toArray(metadatas);
+    if (documents !== undefined) documents = toArray(documents);
```

Comment:
what happens if you try to update with an array of `None`, eg `[None,None,None]`

Reply:
I added an issue to track this - https://github.com/chroma-core/chroma/issues/569